frontend/package.json

```diff
@@ -139,6 +139,7 @@
   },
   "jest": {
     "resetMocks": false,
+    "clearMocks": true,
```

Comment:
force jest to clear mocks between test suites, reduces the risk of a test being introduced that doesn't clean up after itself properly affecting an unrelated test that is executed later.